Example/Tests/ExternallyManagedSessionTest.swift

```diff
@@ -82,7 +82,7 @@ class ExternallyManagedSessionTest: QuickSpec {
                         let sourceConfig = SourceConfig(url: URL(string: "http://a.url")!, type: .hls)
                         sourceConfig.title = "MyTitle"
                         let source = SourceFactory.createSource(from: sourceConfig)
-                        playerDouble.load(source: source)
```

Comment:
Those tests were loading an actual source with doing network requests. This should be prevented in unit test and was changed to use the source stub instead.